docs/bots-guide.md

```diff
@@ -109,12 +109,15 @@ You need:
     * Now, the bot can do its job on the streams you subscribed it to.
     * (In future versions of the API, this step may not be required).
 
-5. Run the bot.
+5. Run the bot. (There are two ways to run your bot.)
 
     * Run
       ```
       zulip-run-bot <bot-name> --config-file ~/.zuliprc
       ```
+      ```
+      zulip-run-bot --path-to-bot <path/to/twibot.py> --config-file ~/.zuliprc
```

Comment:
The command here has actually changed today, `--path-to-bot` won't work any longer on the newst version of the repo. Instead, <bot-name> is either a bot's name or a bot's path.

Reply:
okay

Reply:
@derAnfaenger how about the edit you made on the the part that talks about cloning the python-zulip-api instead of zulip-bots. Is there a change of plan